validator/client/runner.go

```diff
@@ -92,11 +93,14 @@ func run(ctx context.Context, v Validator) {
 						"pubKey": fmt.Sprintf("%#x", bytesutil.Trunc(id[:])),
 						"role":   role,
 					})
+
 					switch role {
 					case pb.ValidatorRole_BOTH:
 						go v.SubmitAttestation(slotCtx, slot, id)
+						go v.SubmitAggregateAndProof(ctx, slot, id)
 						v.ProposeBlock(slotCtx, slot, id)
 					case pb.ValidatorRole_ATTESTER:
+						v.SubmitAggregateAndProof(ctx, slot, id)
```

Comment:
If both of these wait until different times in the slot, you're going to have problems unless they run in goroutines

Reply:
Agree. Switched to go routines